monitor_dashboards.go

```diff
@@ -0,0 +1,115 @@
+package linodego
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/linode/linodego/internal/parseabletime"
+)
+
+// MonitorDashboard represents an ACLP Dashboard object
+type MonitorDashboard struct {
+	ID          int               `json:"id"`
+	Type        DashboardType     `json:"type"`
+	ServiceType ServiceType       `json:"service_type"`
+	Label       string            `json:"label"`
+	Created     *time.Time        `json:"-"`
+	Updated     *time.Time        `json:"-"`
+	Widgets     []DashboardWidget `json:"widgets"`
+}
+
+// enum object for serviceType
+type ServiceType string
+
+const (
+	Linode          ServiceType = "linode"
+	LKE             ServiceType = "lke"
+	DBaaS           ServiceType = "dbaas"
+	ACLB            ServiceType = "aclb"
+	Nodebalancer    ServiceType = "nodebalancer"
+	ObjectStorage   ServiceType = "objectstorage"
+	Vpc             ServiceType = "vpc"
+	FirewallService ServiceType = "firewall"
+)
+
+// enum object for DashboardType
+type DashboardType string
+
+const (
+	Standard DashboardType = "standard"
+	Custom   DashboardType = "custom"
+)
+
+// DashboardWidget represents an ACLP DashboardWidget object
+type DashboardWidget struct {
+	Metric            string            `json:"metric"`
+	Unit              string            `json:"unit"`
+	Label             string            `json:"label"`
+	Color             string            `json:"color"`
+	Size              int               `json:"size"`
+	ChartType         ChartType         `json:"chart_type"`
+	YLabel            string            `json:"y_label"`
+	AggregateFunction AggregateFunction `json:"aggregate_function"`
+}
+
+// Enum object for AggregateFunction
+type AggregateFunction string
+
+const (
+	Min      AggregateFunction = "min"
+	Max      AggregateFunction = "max"
+	Avg      AggregateFunction = "avg"
+	Sum      AggregateFunction = "sum"
+	Rate     AggregateFunction = "rate"
+	Increase AggregateFunction = "increase"
+	Count    AggregateFunction = "count"
+	Last     AggregateFunction = "last"
```

Comment:
Could these names be given more specific names, e.g. `AggregateFunctionMin`?
It's not the prettiest solution but given this is a single-package project we want to be careful to not pollute the namespace with any potential conflicting names 🙂